cosinnus/utils/user.py

```diff
@@ -162,6 +163,40 @@ def get_user_query_filter_for_search_terms(terms):
     return q
         
         
+def create_base_user(email, username=None, password=None):
```

Comment:
@saschan I created a new function here as the `UserCreationForm` from `cosinnus.forms.user` needed additional information like a captcha and I was not sure if I could replace the form. Let me know if it is fine and I will merge both functions.